.github/workflows/template.yml

```diff
@@ -16,6 +16,8 @@ jobs:
     defaults:
       run:
         shell: bash # forces 'Git for Windows' on Windows
+    env:
+      RSPM: 'https://packagemanager.rstudio.com/cran/__linux__/bionic/latest'
```

Comment:
Does this let us use precompiled binaries?

Reply:
yes, so it installs faster